examples_gbs/run_tutorial_points.py

```diff
@@ -0,0 +1,109 @@
+"""
+Point Process Tutorial
+======================
+
+This tutorial shows how to generate GBS point process samples and use them to detect outlier
+points in a data set. Point processes are models for generating random point patterns and GBS
+devices can be programmed to operate as special types of point processes that generate clustered
+point patterns. GBS point processes belong to a class of point processes in which the probability
+of generating a specific pattern of points depends on matrix functions of a kernel matrix that
+describes the similarity between the points. Matrix functions that appear in GBS point processes
+are typically permanents and hafnians. In this tutorial we use the permanental point process in which
+the probability of observing a pattern of points depends on the permanent of their corresponding
+kernel matrix. 
+"""
+
+##############################################################################
+# First we need to import the :mod:`~.gbs.points` and :mod:`~.gbs.plot` modules.
+
+from strawberryfields.gbs import points, plot
+
+##############################################################################
+# We define a space in which the GBS point process patterns are generated. This
+# space is referred to as the state space and is defined by a set of points. The
+# point process selects a subset of these points in each sample. Here we create
+# a discrete and homogeneous two-dimensional space containing 400 points as our
+# state space. 
```

Comment:
I think it would be good to say it in more simple terms: e.g. "Here we create a 20 x 20 square grid of points."